protos/dcs/dcs.proto

```diff
@@ -10,6 +10,7 @@ import "dcs/custom/v0/custom.proto";
 import "dcs/group/v0/group.proto";
 import "dcs/hook/v0/hook.proto";
 import "dcs/mission/v0/mission.proto";
+import "dcs/net/v0/net.proto";
```

Comment:
I agree that it makes sense to keep close to DCS' structure, which is why we should probably move [`DCS.GetMissionName`](https://github.com/DCS-gRPC/rust-server/blob/main/protos/dcs/hook/v0/hook.proto#L7) out into its own namespace eventually, too